files/adfc-hh-connect.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Aufbau einer VPN-Verbindung zum ADFC-HH UCS-MASTER
+
+
+# Uptime & Hostname
+UT=$(uptime | cut -d ' ' -f 4)
```

Comment:
Nicht so optimal:

```
 08:03:24 up 22 min,  4 users,  load average: 0,31, 0,25, 0,30
```
oder 
```
08:04:11 up 108 days, 15:45,  4 users,  load average: 0,26, 0,15, 0,10
```

Reply:
Es gibt sonst noch /proc/uptime

Reply:
Hab's nun über **/proc/uptime** gelöst. Falls nach einer Stunde noch keine Verbindung zustande kam, wird ein Neustart des Clients durchgeführt.

Reply:
Magst du das geänderte Script bitte in den Branch Pushen? Sonst kann ich es mir nicht ansehen.

Reply:
Ist nun online. Danke für das kurze Briefing!

Reply:
Sehe keine Änderung in dieser Zeile